client/me/purchases/purchases-mixin.js

```diff
@@ -6,9 +6,45 @@ import page from 'page';
 /**
  * Internal Dependencies
  */
+import analytics from 'analytics';
 import paths from './paths';
 
 export default {
+	componentWillMount() {
```

Comment:
Using lifecycle methods in `PurchasesMixin` (we don't yet) will make it more difficult to convert this file to a collection of functions in the future, so if we plan to make that change, now might be an ideal time.